kuma/vagrant-wsgi.py

```diff
@@ -0,0 +1,14 @@
+import os
```

Comment:
Sure, can do. Thanks for the feedback.

Is there a name we could use that would differentiate this from *wsgi/kuma.py*? The names are close enough to be confusing. Or is the goal to incorporate *wsgi/kuma.py* into this file over time?